pkgs/by-name/qd/qdiskinfo/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  smartmontools,
+  fetchFromGitHub,
+  cmake,
+  qt6,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "QDiskInfo";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "edisionnano";
+    repo = "QDiskInfo";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-YLNpQOmh/exBJDuYRUEvzNv/El4x+PttEoj+aCur9wg=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qt6.qtbase
+    qt6.qtwayland
+    smartmontools
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE:STRING=MinSizeRel"
+    "-DQT_VERSION_MAJOR=6"
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/QDiskInfo \
+      --prefix PATH : ${smartmontools}/bin
```

Does it expect to see the same version of smartmontools that it saw at build time? If not, this should be suffixed in order to not violate the principle of least surprise.

```suggestion
      --suffix PATH : ${smartmontools}/bin
```